lib/src/kbasesearchengine/events/StatusEvent.java

```diff
@@ -378,13 +400,27 @@ public Builder withNullableNewName(final String newName) {
             }
             return this;
         }
+
+        /** Specify whether the corresponding record for this event should be overwritten
+         * if the record already exists in the index."
+         * @param overwriteExistingData true if the existing record should be overwritten, else false.
+         * @return this builder.
+         */
+        public Builder withOverwriteExistingData(final Boolean overwriteExistingData) {
```

Comment:
I would call the method with withNullableOverwrite... to maintain consistency with the rest of the API and let the user know the boolean may be null.